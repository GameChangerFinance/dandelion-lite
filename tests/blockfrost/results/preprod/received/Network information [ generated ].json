{
  "stake": {
    "active": "377885680591890",
    "live": "333322529882531"
  },
  "supply": {
    "circulating": "30033034780457547",
    "locked": "145258545785872",
    "max": "45000000000000000",
    "reserves": "13566445161364690",
    "total": "31433554838635310",
    "treasury": "1399957359404024"
  }
}
