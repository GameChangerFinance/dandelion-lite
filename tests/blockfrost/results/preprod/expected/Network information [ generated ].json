{
  "stake": {
    "active": "377885680591890",
    "live": "333702725942043"
  },
  "supply": {
    "circulating": "30033036187733663",
    "locked": "145256110252570",
    "max": "45000000000000000",
    "reserves": "13566445161364690",
    "total": "31433554838635310",
    "treasury": "1399957359404024"
  }
}
